pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -1186,6 +1186,7 @@ profiles:
   - args:
       apiVersion: kubescheduler.config.k8s.io/v1beta2
       hardPodAffinityWeight: 5
+      ignorePreferredTermsOfExistingPods: false
```

Comment:
worth adding a test to the scheme tests that sets it to true.

Reply:
Ok I added test cases for both scheme encoding/decoding tests when `ignorePreferredTermsOfExistingPods` is true.